File: flutter/lib/models/round_score.dart
class RoundScore {
  final int? id;
  final String sessionId;
  final int roundNumber;
  final String playerName;
  final int points;
  final DateTime createdAt;

  /// Ligretto only: number of cards left in the Ligretto pile for this round.
  final int? remainingCards;

  /// Ligretto only: number of cards placed in the middle for this round.
  final int? placedCards;

  const RoundScore({
    this.id,
    required this.sessionId,
    required this.roundNumber,
    required this.playerName,
    required this.points,
    required this.createdAt,
    this.remainingCards,
    this.placedCards,
  });
}

File: DaPoint/RoundScore.swift
import Foundation
import SwiftData

@Model
final class RoundScore {
    var sessionId: UUID
    var roundNumber: Int
    var playerName: String
    var points: Int

    init(sessionId: UUID, roundNumber: Int, playerName: String, points: Int) {
        self.sessionId = sessionId
        self.roundNumber = roundNumber
        self.playerName = playerName
        self.points = points
    }
}
